### PR TITLE
Add support for Type 5 as now mandated by NFC Forum.

### PR DESCRIPTION
**Pull request by collador**
Adding type 5 to specs.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -584,6 +584,13 @@
         service. Supports three different communication speeds 106 or 212 or
         424 Kbits/s.
       </li>
+      <li><b>NFC Forum Type 5</b>: This tag is based on the ISO/IEC 15693
+        (NFC-V) standard that supports anti-collision protection and multi tag
+        addressing. Like Type 4 and Type 3, Type 5 tags can be configured to be
+        either rewritable or read-only. Like Type 3 memory size can vary, with
+        a theoretical memory limit is 2MBytes. Communication speed is 26 Kbits/s.
+        Supports long range distance.
+      </li>
```
Comment (zolkis) [inline index.html:593]: It would make sense to indicate the range in this summary.

**Comment by anssiko** [comment]
@collador Thanks for the PR. Before we can merge your proposed changes, per [CONTRIBUTING][1], you must join the [W3C Web NFC Community Group][2]. I've sent you further information on how to join.

[1]: https://github.com/w3c/web-nfc/blob/gh-pages/CONTRIBUTING.md
[2]: https://www.w3.org/community/web-nfc/

**Comment by kenchris** [comment]
Any update @collador ?

**Comment by anssiko** [comment]
@collador seems inactive, so I'll close this PR.

The issue #133 is still in scope of the group, so the group's active participants can look into the issue.

**Pull request closed without merging**